docs/pages/machine-workload-identity/mwi-use-cases/mwi-use-cases.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Machine & Workload Identity Use Cases
+description: Common use cases with Machine ID & Workload Identity
+labels:
+ - conceptual
+ - mwi
+---
+
+Machine & Workload Identity can work together to create a comprehensive security model. Machines can securely access resources while workloads communicate securely with each other and external services, 
+all managed through Teleport's infrastructure identity platform.
+
+## CI/CD pipeline with end-to-end authentication
+
+A [CI/CD system](../mwi-use-cases/mwi-ci-cd.mdx) securely deploys services to Kubernetes and establishes secure communication channels between them.
+The pipeline authenticates through the [Teleport Proxy Service](../../reference/architecture/architecture.mdx#teleport-proxy-service) to deploy to Kubernetes and receives credentials to interact with cloud APIs (e.g., to push container images)
+
+## Cloud-native application with third-party API access
+
+These examples illustrate how Machine & Workload Identity enable secure, scalable API access for cloud-native applications running in Kubernetes:
+
+- Monitoring and observability exporters
```

Comment:
Are all of these meant to be a nested bullet, or have the initial part like "Monitoring and observability exports" bolded or otherwise differentiated? They're hard to parse on the preview site.